src/kapil/Assi30/SecondMax.java

```diff
@@ -0,0 +1,23 @@
+package kapil.Assi30;
+
+public class SecondMax {
+
+	void getSecondMax(int[] arr) {
+		int max = arr[0];
+		int secondmx = arr[1];
+		for (int index = 1; index < arr.length; index++) {
+			if (max < arr[index]) {
```

Comment:
this fails for below input
int input[] = {10, 33, 43, 55, 97, 111, 99, 3};

Reply:
for (int index = 0; index < input.length; index++) {

            if (max < input[index]) {
                secondMax = max;
                max = input[index];
            } else if (input[index] > secondMax && max != input[index]) {
                secondMax = input[index];
            }
        }